docs/swagger/spec.yml

```diff
@@ -20,8 +20,12 @@ consumes:
 tags:
```

Comment:
It should, but maybe we actually don't need it to. The same parameters for scenes are parsed and passed into `listScenes` for a bucket.